File: Nov2020/27Nov/BobAndString.cpp
/*
Problem Name: Bob and String
Problem Link: https://www.hackerearth.com/practice/data-structures/hash-tables/basics-of-hash-tables/practice-problems/algorithm/bob-and-string-easy/
*/

#include<bits/stdc++.h>
using namespace std;
int main(){
    ios::sync_with_stdio(0);    
    cin.tie(0);             
    int t;
    cin>>t;
    while(t--){
        string s,t;
        cin>>s>>t;
        int s1[26]={0}, t1[26]={0};
        for(int i=0;i<s.length();++i)
            s1[int(s[i])-'a']++;
        
        for(int i=0;i<t.length();++i)
            t1[int(t[i])-'a']++;
        
        int op=0;
        for(int i=0;i<26;++i){
            op+=abs(s1[i]-t1[i]);
        }
        cout<<op<<"\n";
    }
    return 0;
}

File: Nov2020/27Nov/BobAndString.java
/*
Problem Name: Bob and String
Problem Link: https://www.hackerearth.com/practice/data-structures/hash-tables/basics-of-hash-tables/practice-problems/algorithm/bob-and-string-easy/
*/
import java.util.*;
class BobAndString {
    public static void main(String args[] ) throws Exception {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        for(;n>0;n--){
            String s = in.next(), t = in.next();
            int sarr[] = new int[26], tarr[] = new int[26];
            for(char c: s.toCharArray())
                sarr[c-'a']++;
            for(char c: t.toCharArray())
                tarr[c-'a']++;
            int operations = 0;
            for(int i=0;i<26;i++){
                operations += Math.abs(sarr[i] - tarr[i]);
            }
            System.out.println(operations);
        }
    }
}
